src/java.desktop/share/classes/javax/swing/JList.java

```diff
@@ -2507,8 +2507,7 @@ else if (getModel().getSize() > 0) {
      *                    {@code SwingConstants.VERTICAL}
      * @param direction less or equal to zero to scroll up/back,
      *                  greater than zero for down/forward
-     * @return the "unit" increment for scrolling in the specified direction;
-     *         returns a non-negative value
+     * @return the non-negative "unit" increment value for scrolling in the specified direction
```

Comment:
I guess the length is exceeding 80 column.

Reply:
updated